projects/storefrontlib/recipes/config/layout-config.ts

```diff
@@ -126,3 +130,32 @@ export const layoutConfig: LayoutConfig = {
     },
   },
 };
+
+/**
+ * Factory for layout configuration.
+ */
+export function layoutConfigFactory(): LayoutConfig {
+  const config: LayoutConfig = JSON.parse(JSON.stringify(layoutConfig));
+  const unifiedDefaultHeaderSlotsAcrossBreakpoints =
+    inject(FeatureToggles).unifiedDefaultHeaderSlotsAcrossBreakpoints;
+
+  if (
+    unifiedDefaultHeaderSlotsAcrossBreakpoints &&
+    config.layoutSlots &&
+    config.layoutSlots.header &&
+    'slots' in config.layoutSlots.header
+  ) {
+    config.layoutSlots.header.slots = [
+      'PreHeader',
+      'SiteContext',
+      'SiteLinks',
+      'SiteLogo',
+      'SearchBox',
+      'SiteLogin',
+      'MiniCart',
+      'NavigationBar',
+    ];
+  }
```

Comment:
I'm aware we're already after a few rounds of review. Forgive me please that-late observation. I can see that we can also remove the `lg`-breakpoint specific config:

```suggestion
  if (
    unifiedDefaultHeaderSlotsAcrossBreakpoints &&
    config.layoutSlots &&
    config.layoutSlots.header &&
    'slots' in config.layoutSlots.header
  ) {
    if ('lg' in config.layoutSlots.header) {
      delete config.layoutSlots.header.lg;
    }
    
    config.layoutSlots.header.slots = [
      'PreHeader',
      'SiteContext',
      'SiteLinks',
      'SiteLogo',
      'SearchBox',
      'SiteLogin',
      'MiniCart',
      'NavigationBar',
    ];
  }
```

See original config above:
<img width="361" height="385" alt="Image" src="https://github.com/user-attachments/assets/a4dc9edd-c672-4cd2-9f20-5c91dff20b12" />